ta/os_test/os_test.c

```diff
@@ -902,9 +901,8 @@ TEE_Result ta_entry_client_with_timeout(uint32_t param_types,
 				&ret_orig);
 
 	if (ret_orig != TEE_ORIGIN_TRUSTED_APP || res != TEE_ERROR_CANCEL) {
-		EMSG("ta_entry_client_with_timeout: TEE_InvokeTACommand: "
-		     "res 0x%x ret_orig 0x%x\n", (unsigned int)res,
-		     (unsigned int)ret_orig);
+		EMSG("TEE_InvokeTACommand: res 0x%x ret_orig 0x%x\n",
+		     (unsigned int)res, (unsigned int)ret_orig);
```

Comment:
While you're at it, maybe you can further cleanup these message traces: remove `\n` terminal character (here and there) and use here `PRIx32` specifier instead of `"x"` for `res` and `ret_orig` to avoid the cast.

However, since it's also out of the scope of  your initial P-R, you can discard my comment, we'll address that with another P-R.

Reply:
Follow-up [here](https://github.com/OP-TEE/optee_test/pull/709/commits/ec21b57c5123a4ed4cd0ead5a29a9e72351c951d).

Reply:
> I you want to apply, please do it on top of this branch, I do not want to rebase (too lazy).
> 
> For the PRIx32, I will let someone else take care of it :)

Never mind, I'll take care. Thanks for the hints, they remind me unix-like shell commands are sometimes pretty or ugly depending on the viewer's eyes.